main.go

```diff
@@ -31,6 +33,8 @@ type importBody struct {
 
 const buildDefinitionFileName = ".wharf-ci.yml"
 
+var log = logger.NewScoped("WHARF-PROVIDER-GITHUB")
```

Comment:
(nit) Magic string here could be defined as const.

Reply:
What variable name do you propose?

In my mind that would be something like

```go
const loggerScopeName = "WHARF-PROVIDER-GITHUB"

var log = logger.NewScoped(loggerScopeName)
```

which is kind of double naming. Don't see the need for extracting it here.